packages/sth-config/src/config-service.ts

```diff
@@ -83,4 +83,17 @@ export class ConfigService {
     update(config: DeepPartial<STHConfiguration>) {
         merge(this.config, config);
     }
+
+    static getConfigInfo(config: STHConfiguration): PublicSTHConfiguration {
+        const {
+            kubernetes: kubeFull,
+            sequencesRoot: optionsSequencesRoot2,
```

Comment:
shouldn't this be `optionsSequencesRoot`?

Reply:
[According to Michał](https://scramjetcloudplatform.slack.com/archives/C02FC09DNUV/p1648478075077029?thread_ts=1648470345.103749&cid=C02FC09DNUV) there are two independent dummy variables, so this is ok